src_assets/common/assets/web/apps.html

```diff
@@ -209,21 +205,21 @@ <h1>{{ $t('apps.applications_title') }}</h1>
         </div>
         <!-- elevation -->
         <div class="mb-3 form-check" v-if="platform === 'windows'">
-          <label for="appElevation" class="form-check-label">{{ $t('_common.run_as') }}</label>
+          <label for="appElevation" class="form-check-label">{{ $t('_common.run_as') }}<div class="mt-0 form-text">{{ $t('_common.disabled_def_cbox') }}</div></label>
```

Comment:
```suggestion
          <label for="appElevation" class="form-check-label">{{ $t('_common.run_as') }}
            <div class="mt-0 form-text">{{ $t('_common.disabled_def_cbox') }}</div>
          </label>
```

Can we try to avoid nested elements on a single line?